operator/pkg/controlplane/controlplane.go

```diff
@@ -24,33 +25,49 @@ func EnsureControlPlaneComponent(component, name, namespace string, client clien
 
 	deployment, ok := deployments[component]
 	if !ok {
-		return fmt.Errorf("no exist manifest for %s", component)
+		klog.V(2).InfoS("skip applying karmada component", "component", component, "karmada", klog.KRef(namespace, name))
```

Comment:
This log is a little strange.

Reply:
How about  `Skip installing karmada componnet %s", component, "karmada", klog.KRef(namespace, name)`

Reply:
Can you give a example about this log?

Reply:
example for karmada descheduler, if it's cfg is nil, will log the message: `skip installing component karmada-descheduler`.